.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
```

Comment:
A third issue template for support requests would be nice, like Gutenberg does it: https://github.com/WordPress/gutenberg/blob/4b3324b1d8db6ef6da3866c772843889d1182612/.github/ISSUE_TEMPLATE/Custom.md

Reply:
Actually we are trying as much as possible to redirect support to the [AMP Plugin .org support forum](https://wordpress.org/support/plugin/amp/) to having providing support on GitHub and on .org. In fact, we should have a canned response to ask users posting support request on GitHub to channel their request on .org

Reply:
That's why I suggested using a template similar to Gutenberg's. People will then click on "Help request" and read the template which says "please go to the forums". Otherwise I fear people will just pick "Bug report", remove all the contents and then post their support question that way.

Reply:
Ah, my bad! Yep will add that shortly!